tools/transformer/src/transform.js

```diff
@@ -1,29 +1,21 @@
-const { readFileSync, writeFileSync } = require("fs");
 const {
   applyAllOfTransformations,
   applyOneOfTransformations,
   applyModelNameTransformations,
 } = require("./transformations");
-const { parse, stringify } = require("yaml");
+const { getAPI, saveAPI } = require("./apifile");
 
-let apiFileLocation = process.argv.slice(2);
-console.error(apiFileLocation);
-if (!apiFileLocation && !apiFileLocation[0]) {
-  throw new Error("Missing positional argument for openapi file");
-}
+const log = require("simple-node-logger").createSimpleLogger();
```

Comment:
This one guarantees compatibility with the console format so we can reuse the same interface and swap to anything else we need.